src/styles/color.ts

```diff
@@ -1,5 +1,6 @@
 const color = {
   MAIN: '#7782FF',
+  MAIN_2: '#9DB7FF',
```

Comment:
```suggestion
  SUB: '#9DB7FF',
```
또는
```suggestion
  SECONDARY: '#9DB7FF',
```
로 위계를 표현해도 좋아보여요

Reply:
MAIN과 비슷한 색상이라 _2로 작성했는데 SECONDARY가 더 직관적이고 좋은 것 같네요!!!! 
[ffa5a8f](https://github.com/JECT-Study/PICK-O-Client/pull/310/commits/ffa5a8f05947b8ef5cd0876ad967eb1f68b6dbc2) 수정했습니다!!